websites/P/Pixelmon Mod/presence.ts

```diff
@@ -0,0 +1,63 @@
+const presence = new Presence({
+		clientId: "976435781486911509"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	page = document.location.href;
+
+let title: Element, title2: Element;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo"
+	};
+	presenceData.startTimestamp = browsingTimestamp;
+	title = document.querySelector(
+		"#page-body > main > div > div.col-md-9 > div.side-segment > h3 > a"
+	);
+	title2 = document.querySelector(
+		"#page-body > main > div > div.col-md-9 > div.side-segment > h3"
+	);
+	const search = document.querySelector<HTMLInputElement>("#searchInput");
+	if (page.includes("news.php") || document.location.pathname == "/")
+		presenceData.details = "Homepage";
+	else if (page.includes("/wiki/") || page.includes("/w/index.php?")) {
+		const firstChild = document.querySelector(
+			"#content > div.contentHeader"
+		)?.firstElementChild;
+		if (search.value) {
+			presenceData.smallImageKey = "search";
+			presenceData.details = "Searching for:";
+			presenceData.state = search.value;
+		} else if (firstChild.className.includes("firstHeading")) {
+			presenceData.state = firstChild.textContent;
+			presenceData.smallImageKey = "reading";
+			presenceData.details = "Reading Wiki:";
+		} else if (document.querySelector("#firstHeading-h2csdq87lb"))
+			presenceData.details = "Search Results";
+		else presenceData.details = "Reading Wiki:";
+	} else if (page.includes("index.php")) presenceData.details = "Forum";
+	else if (page.includes("viewforum.php"))
+		presenceData.details = `${title.textContent} Forum`;
+	else if (page.includes("viewtopic.php")) {
+		presenceData.details = "Viewing Post:";
+		presenceData.state = title.textContent;
+	} else if (page.includes("team.php")) {
+		presenceData.details = `Pixelmon's ${
+			document.querySelector("li[class='team-nav-active']").textContent
+		} Team`;
+	} else if (page.includes("ucp.php?mode=")) {
+		presenceData.details = title2.textContent;
+		presenceData.smallImageKey = "reading";
+	} else if (page.includes("downloads.php")) presenceData.details = "Downloads";
+	else if (page.includes("donate.php")) presenceData.details = "Donate";
```

Comment:
please elaborate on these names

Reply:
Downloads as in you can download everything here https://pixelmonmod.com/downloads.php
Donate as in you can donate here https://pixelmonmod.com/donate.php
_Just using the same names as on the site_

Reply:
I understood, I’m saying you should say a bit more than just a word